modules/ROOT/pages/creating-and-managing-a-cluster-manually.adoc

```diff
@@ -4,27 +4,19 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: cluster, deploy
 
-*_Enterprise Edition_*
-
 This page describes manual creation and configuration of a cluster. There are two ways to create and manage clusters:
 
 * Using Runtime Manager
 +
 See xref:runtime-manager::managing-servers.adoc#create-a-cluster[Creating and Managing Clusters] for configuration instructions.
 * Manually, using a configuration file
 
+== Considerations
 
-[CAUTION]
-====
-*Do not attempt mixed management of clusters*
-
-If you create a cluster manually, do not attempt to manage it via the Management Console. The Management Console cannot recognize your manually-created cluster, and overwrites your cluster configuration.
-====
-
-[NOTE]
-====
-All nodes in a cluster must have the same agent version.
-====
+* Do not mix cluster management tools
++
+Manual cluster configuration is not synced to Anypoint Runtime Manager, so any change you make in the platform overrides the cluster configuration files. To avoid this scenario, ensure you use only one method for creating and managing your clusters: either manual configuration or using Anypoint Runtime Manager.
```

Comment:
```suggestion
Manual cluster configuration is not synced to Anypoint Runtime Manager, so any change you make in the platform overrides the cluster configuration files. To avoid this scenario, use only one method for creating and managing your clusters: either manual configuration or configuration using Anypoint Runtime Manager.
```